extensions/wikia/CommunityPage/models/CommunityPageSpecialInsightsModel.class.php

```diff
@@ -74,15 +70,10 @@ private function getInsightModule( $type, $config ) {
 		 *
 		 * communitypage-popularpages-title'
 		 * communitypage-uncategorizedpages-title'
-		 * communitypage-wantedpages-title'
 		 * communitypage-deadendpages-title'
-		 * communitypage-wantedpages-description'
 		 */
 		$insightPages['type'] = $type;
 		$insightPages['title'] = wfMessage( 'communitypage-' . $type . '-title' )->text();
-		$insightPages['description'] =  $type === 'wantedpages' ?
-			wfMessage( 'communitypage-wantedpages-description' )->text() :
-			null;
 
 		if ( $insightPages['count'] > self::INSIGHTS_MODULE_ITEMS ) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)